source/_drafts/videojs-vr-now-under-the-video-js-org.md

```diff
@@ -0,0 +1,57 @@
+
+The first and default setting for `projection` is `'AUTO'`. Setting `projection` to `'AUTO'` tells videojs-vr to look at `player.mediainfo.projection`. `player.mediainfo.projection` will have to be set by some external plugin/script which is told by a sever that the current video is 360/VR. The `player.mediainfo.projection` of a video can be any of the following:
+
+* `'360'`, `'Sphere'`, or `'equirectangular'`: The video is a sphere
+* `'Cube'` or `'360_CUBE'`: The video is a cube
+* `'NONE'`: This video is not a 360 video, the videojs-vr plugin should do nothing. this would not have to be set as it will be assumed if no projection exists.
+* `'360_LR'`: Used for side-by-side 360 videos
+* `'360_TB'`: Used for top-to-bottom 360 videos
+
+Otherwise, the `projection` can manually be set on plugin intialization to any of the above values. The plugin can then be disposed and re-intialized for each video with a different setting.
+
+## What happens after after we know a video is 360/VR?
+Using the current projection value we determine what we need to show on our [three.js](https://github.com/mrdoob/three.js) canvas. From there we use the polyfilled (via [webvr-polyfill](https://github.com/googlevr/webvr-polyfill) or real [VRDisplay API](https://developer.mozilla.org/en-US/docs/Web/API/VRDisplay) to set up controls. If the device has a VIVE, Oculus, etc plugged in we use that. If not we display the video in 360 mode where the mouse/keyboard/accelerometer control the video. On mobile devies there is also a button to go into cardboard mode which will show the video as two smaller videos separated by a line. This is so that the video can be viewed using pseudo VR headsets (such as google cardboard) and a phone. While the video is playing we use animation frames to animate the VideoTexture on the three.js canvas. That allows us to show the correct portion of the video based on the controls that the user is inputting.
+
+## Why is it so huge?
+We all agree that the plugin is huge, but due to the fact that browsers do not all natively support the VRDisplay API, we have to include a lot of big dependencies in our project. Mainly the size comes from the following:
+
+1. [three.js](https://github.com/mrdoob/three.js) which we use to create the VR/360 canvas, and implementing it on our own would be painful
+2. [webvr-polyfill](https://github.com/googlevr/webvr-polyfill) which polyfills VR features for browsers that don't currently support VR (outside of experimental flags and builds)
+3. [webvr-boilerplate](https://github.com/borismus/webvr-boilerplate) this was used to get the plugin off of the ground quicker but will probably be replaced in the future with a smaller in house implementation. It basically does a bit more polyfill for us.
+
+## Debugging
+Every object from three.js and webvr manager (from webvr-boilerplate) are available on the instance of the VR plugin that will is returned by `player.vr()`.
```

Comment:
Should be `Every object ... is available` instead of `Every object ... are available`.